# This config file is used by ros2_control
controller_manager:
  ros__parameters:
    update_rate: 500  # Hz

    arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController
    gripper_controller:
      type: joint_trajectory_controller/JointTrajectoryController
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

arm_controller:
  ros__parameters:
    joints:
      - joint1
      - joint2
      - joint3
      - joint4
      - joint5
      - joint6
    command_interfaces:
      - position
      - velocity
    state_interfaces:
      - position
      - velocity
gripper_controller:
  ros__parameters:
    joints:
      - joint7
      - joint8
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity

    # gains:
    #   joint7: {p: 80.0, i: 0.1, d: 0.0, i_clamp: 3.0}
    #   joint8: {p: 80.0, i: 0.1, d: 0.0, i_clamp: 3.0}